Dockerfile

```
@@ -1,10 +1,11 @@
 FROM nginx
-MAINTAINER Ash Wilson <smashwilson@gmail.com>
+LABEL Ash Wilson <smashwilson@gmail.com>
```

```suggestion
LABEL maintainer=Ash Wilson <smashwilson@gmail.com>
```

Following the convention suggested from [the `Dockerfile` docs](https://docs.docker.com/engine/reference/builder/#maintainer-deprecated).

It'd be nice to get other maintainers credited in here too 🤔